src/components/Grain.js

```diff
@@ -18,7 +19,11 @@ const Grain = ({ grain, farm, stub }) => {
 
   const navigate = useNavigate();
 
-  const [deleteGrain, { data, loading, error }] = useMutation(DELETE_GRAIN);
+  const [deleteGrain, { data, loading, error }] = useMutation(DELETE_GRAIN, {
```

Comment:
Thank you for adding this -- I can't believe I forgot it when I did the other refetches!